backend/src/main/java/org/eclipse/tractusx/puris/backend/masterdata/domain/model/Partner.java

```diff
@@ -47,20 +48,26 @@
 @NoArgsConstructor
 public class Partner {
 
+    public final static String BPNL_REGEX = "^BPNL[0-9a-zA-Z]{12}$";
+    public final static String EDC_REGEX = "^http[s]?://([a-z0-9][a-z0-9\\-]+[a-z0-9])(\\.[a-z0-9\\-]+)*(:[0-9]{1,4})?(/[a-z0-9\\-]+)+[/]?$";
```

Comment:
The hostname regex here does not include the following hostname: "https://isst-edc.int.demo.catena-x.net/management" which is used on integration test ingress.

I propose following regex: ^(?:\w+://)?((?:[^./?#]+\.)?([^/?#]+))(/\w+)*